app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -90,10 +102,10 @@ object AppRepository {
     private lateinit var scheduleNetworkRepository: ScheduleNetworkRepository
     private lateinit var engelsystemNetworkRepository: EngelsystemNetworkRepository
     private lateinit var sharedPreferencesRepository: SharedPreferencesRepository
+    private lateinit var sessionsTransformer: SessionsTransformer
 
-    private var onSessionsChangeListener: OnSessionsChangeListener? = null
-    private var alarmsHaveChanged = false
-    private var highlightsHaveChanged = false
+    private val mutableLoadScheduleState = MutableSharedFlow<LoadScheduleState>(extraBufferCapacity = 1)
+    val loadScheduleState: SharedFlow<LoadScheduleState> = mutableLoadScheduleState
```

Comment:
Why do you think so?

Reply:
Effectively it is an alias of  `mutableLoadScheduleState` (which is private) as (Unmutable)SharedFlow...exposing private properties is mostly done via getters not by exposing them directly via props. And the receiver will not even notice the difference.

Reply:
So you want to change the code to this?

```kotlin
val loadScheduleState: SharedFlow<LoadScheduleState>
    get() = mutableLoadScheduleState
```

It doesn't make much of a difference. In fact the generated bytecode is almost the same (a `getLoadScheduleState` method is used under the hood in both cases).

Reply:
Exactly! And even better when compiler already does it, then it makes also visible for the devs.

Reply:
@bitPogo: So you like it better. But you have failed to mention **why** that is. What's the advantage you see in one version over the other? From reading your comments it feels like it comes down to "it doesn't look familiar to me". Given that @johnjohndoe is doing most of the work in this code base, I'm not sure that's enough of a reason to change the code :smile:

Reply:
:x: I won't use an explicit getter here which is intended for reloading a value which should not be cached. Here it is a stream to be exposed. Therefore, I believe the getter makes no sense.

Reply:
What I was expecting as equivalent was something like: 
``` java
public class Example {
    private List<String> a = Collections.singletonList("...");
    final public ArrayList<String> b = (ArrayList<String>) a;
}
```
rather than:
``` java
public class Example {
    private List<String> a = Collections.singletonList("...");
    ArrayList<String> getB() {
        return (ArrayList<String>) a;
    }
}
```

However in general, what I was not aware of...there or no Properties like in Java or Rust or you name it...there are only getters/setters in Kotlin...